src/workloads/scale/LLTInsert.yml

```diff
@@ -25,22 +25,31 @@ GlobalDefaults:
   # Loader Config.
   LoadThreads: &LoadThreads 4
   LoadBatchSize: &LoadBatchSize 1000
+
+  # In-memory
+  InitialDocumentCount: &InitialNumDocs 10000000
```

Comment:
Could you include a comment about the expected dataset size based on these numbers. Whether it is in memory or out of memory will depend on that and the system it is running on.